code/game/objects/items/mountable_frames/newscaster_frame.dm

```diff
@@ -4,13 +4,51 @@
 	icon_state = "newscaster"
 	item_state = "syringe_kit"
 
-	materials = list(MAT_METAL=14000, MAT_GLASS=8000)
+	materials = list(MAT_METAL=6000, MAT_GLASS=2000)
 	mount_requirements = MOUNTED_FRAME_SIMFLOOR | MOUNTED_FRAME_NOSPACE
-	metal_sheets_refunded = 7
-	glass_sheets_refunded = 4
+	metal_sheets_refunded = 3
+	glass_sheets_refunded = 1
 
 /obj/item/mounted/frame/newscaster_frame/do_build(turf/on_wall, mob/user)
 	var/obj/machinery/newscaster/N = new /obj/machinery/newscaster(get_turf(src), get_dir(on_wall, user), 1)
 	N.pixel_y -= (loc.y - on_wall.y) * 32
 	N.pixel_x -= (loc.x - on_wall.x) * 32
 	qdel(src)
+/obj/item/mounted/frame/newscaster_frame/display_frame
+	name = "status display frame"
+	desc = "Used to build status displays, just secure to the wall."
+	icon = 'icons/obj/status_display.dmi'
+	icon_state = "frame"
+	item_state = "syringe_kit"
+
+/obj/item/mounted/frame/newscaster_frame/display_frame/do_build(turf/on_wall, mob/user)
+	var/obj/machinery/status_display/N = new /obj/machinery/status_display(get_turf(src), get_dir(on_wall, user), 1)
+	N.pixel_y -= (loc.y - on_wall.y) * 32
+	N.pixel_x -= (loc.x - on_wall.x) * 32
+	qdel(src)
+
+/obj/item/mounted/frame/newscaster_frame/ai_display_frame
+	name = "ai status display frame"
+	desc = "Used to build ai status displays, just secure to the wall."
+	icon = 'icons/obj/status_display.dmi'
+	icon_state = "frame"
+	item_state = "syringe_kit"
+
+/obj/item/mounted/frame/newscaster_frame/ai_display_frame/do_build(turf/on_wall, mob/user)
+	var/obj/machinery/ai_status_display/N = new /obj/machinery/ai_status_display(get_turf(src), get_dir(on_wall, user), 1)
```

Comment:
Any reason why you pass 2 more arguments for these? I don't see any arguments being expected in either of the paths here. I see that `newscaster` also does it, but again, `/obj/machinery/newscaster/Initialize` only expects `mapload` which is passed by default to any `Initialize` call.

Reply:
they are being used..no?

Reply:
They are not, that's why we have to pixel bump and rotate `N` in the lines that follow this